### ISSUE TITLE
Accessibility: Removing `aria-label` set to empty string in all switches and an issue with announcing labels when moving using arrow keys

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#18111]
### Describe the bug

Reported issue details: The enabled checkbox has an empty aria-label (e. g. explicitly set to an empty string) and an aria-labelledby attribute. No idea whether that is the reason for the behavior, but the label is not announced in Firefox when moving using the arrow keys in browse mode, but is announced when moving using the tab key in whatever mode. This behavior unfortunately applies to all checkboxes throughout the admin interface. 

The tester was using NVDA Alpha 26228 tool.

Testing device details: 
* Windows 10, build 19044.1889
* Firefox 103.0.2 or Microsoft Edge 104.0.1293.54

Note: I have investigated this issue on MacbookPro, Chrome browser and ChromeVox extension and while I was able to see the empty `aria-label` I was not able to replicate the label not being announced when moving using the arrow keys This is because I don't know how the tester was using the arrow keys to navigate (it could be specific to NVDA tool I'm not able to use on MacbookPro, I will get more info upon meeting with the tester and will update this issue).

![Screenshot 2022-08-24 at 13 25 22](https://user-images.githubusercontent.com/4890675/186418380-abf1757b-11ee-4b8c-a870-a7996559d9fb.png)

### Version

19.0.0

### Expected behavior

The labels for switches should be announced in Firefox when moving using the arrow keys.

### Actual behavior

The labels for switches are not announced in Firefox when moving using the arrow keys.

### How to Reproduce?

- Switch to the newly created realm using the dropdown in the left-side menu
- Click on the Realm settings in the left-side menu
- Click on the hint button (question mark in a circle) at one of the fields in the General tab i.e Frontend URL
